addon/-private/data-view/radar/radar.js

```diff
@@ -307,12 +322,16 @@ export default class Radar {
     this.scheduleUpdate();
   }
 
-  resetItems(items) {
+  updateItems(items, isReset = false) {
```

Comment:
Renaming this function has implications for DynamicRadar, it overrides the `resetItems` function and calls super. We either need to roll that logic into here somehow, or we need to rethink the reset logic.